File: TwoPointers/js/uncompress.js
const uncompress=(s)=>{
    let i = 0
    let j = 0
    const result = []
    const numbers = '0123456789'

    while(j < s.length){
        if(numbers.includes(s[j])){
            j++
        }else{
            const count = Number(s.slice(i, j))

            for(let k = 0; k<count; k++){
                result.push(s[j])
            }
            j +=1
            i = j
        }
    }

    return result.join('')
}

File: TwoPointers/python/uncompress.py
def uncompress(s: str) -> str:
    result: list[str] = []
    i = 0
    while i < len(s):
        j = i
        while j < len(s) and s[j].isdigit():
            j += 1
        count = int(s[i:j])
        result.append(s[j] * count)
        i = j + 1
    return "".join(result)
